app/views/admin/conferences/edit.html.haml

```diff
@@ -22,11 +22,11 @@
         = f.input :timezone, as: :time_zone, hint: 'The conference time zone'
         = f.input :start_date, as: :string, input_html: { id: 'conference-start-datepicker', readonly: 'readonly' }
         = f.input :end_date, as: :string, input_html: { id: 'conference-end-datepicker', readonly: 'readonly' }
-        = f.input :start_hour, input_html: {size: 2, type: 'number', min: 0, max: 23}
-        = f.input :end_hour, input_html: {size: 2, type: 'number', min: 1, max: 24}
+        = f.input :start_hour, input_html: { size: 2, type: 'number', min: 0, max: 23}, hint: 'Changing the conference hours will unschedule your scheduled events.'
```

Comment:
Since you fixed styling for the first curly quote, please add a space before the last one as well. Same goes for the next line, as well as for L32

Reply:
ok 😄